posts/2023-XX-XX-rustfmt-supports-let-else-statements.md

```diff
@@ -0,0 +1,62 @@
+
+The initial Style Guide was developed many years ago (beginning in 2016), and was driven by a Style Team in collaboration with the community through an RFC process. The Style Guide was then made official in 2018 via [RFC 2436].
+
+That initial Style Team was more akin to a Project Working Group in today's terms, as they had a fixed scope with a main goal to simply pull together the initial Style Guide. Accordingly that initial Style Team was disbanded once the Guide was made official.
+
+There was subsequently no designated group within the Rust Project that was 
+explicitly responsible for the Style Guide, and no group explicitly focused on determining the official Style for new language constructs.
+
+The absence of a team/group with ownership of the Style Guide didn't really cause problems at first, as the new syntax that came along during the first few years was comparatively non-controversial when it came to default style and formatting. However, over time challenges started to develop when there was increasingly less community consensus and no governing team within the Project to make the final decision about how new language syntax should be styled.
+
+This was certainly the case with let-else statements, with lots of varying perspectives on how they should be styled. Without any team/group to make the decision and update the Style Guide with the official rules for let-else statements, Rustfmt was blocked and was unable to proceed.
+
+These circumstances around let-else statements resulted in a greater understanding across the Project of the need to establish a team to own and maintain the Style Guide. However, it was also well understood that spinning up a new team and respective processes would take some time, and the decision was made to not block the stabilization of features that were otherwise fully ready to be stabilized, like let-else statements, in the nascency of such a new team and new processes.
+
+Accordingly, let-else statements were stabilized and released without formatting support and with an understanding that the new Style Team and then subsequently the Rustfmt Team would later complete the requisite work required to incorporate formatting support.
+
+## Steps Taken
+
+A number of steps have been taken to improve matters in this space. This includes steps to address the aforementioned issues and deal with some of the "style debt" that accrued over the years in the absence of a Style Team, and also to establish new processes and mechanisms to bring about other formatting/styling improvements.
+
+* [Launched a new, permanent Style Team][style-team-rfc] that's responsible for the Style Guide.
+* Established a mechanism to evolve the default style while still maintaining stability guarantees ([RFC 3338][style-edition-rfc]).
+* Developed a [nightly-syntax-policy] that provides clarity around style rules for unstable/nightly-only syntax, and enables Rustfmt to provide earlier support for such syntax.
```

Comment:
I know we haven't landed the two PRs that establish this, though we have universal agreement and all the FCP boxes are checked so I thought it would be reasonable to go ahead and mention this as a "done" item